docs/de/docs/fastapi-people.md

```diff
@@ -0,0 +1,171 @@
+
+{% if people %}
+<div class="user-list user-list-center">
+{% for user in people.last_month_active %}
+
+<div class="user"><a href="{{ user.url }}" target="_blank"><div class="avatar-wrapper"><img src="{{ user.avatarUrl }}"/></div><div class="title">@{{ user.login }}</div></a> <div class="count">Issues replied: {{ user.count }}</div></div>
+{% endfor %}
+
+</div>
+{% endif %}
+
+## Experten
+
+Hier sind die **FastAPI-Experten**. 🤓
+
+Dies sind die Benutzer, die [anderen am meisten mit Problemen (Fragen) auf GitHub geholfen haben](help-fastapi.md#help-others-with-issues-in-github){.internal-link target=_blank} durch *alle Zeiten*.
+
+Sie haben bewiesen, dass sie Experten sind, indem sie vielen anderen geholfen haben. ✨
+
+{% if people %}
+<div class="user-list user-list-center">
+{% for user in people.experts %}
+
+<div class="user"><a href="{{ user.url }}" target="_blank"><div class="avatar-wrapper"><img src="{{ user.avatarUrl }}"/></div><div class="title">@{{ user.login }}</div></a> <div class="count">Issues replied: {{ user.count }}</div></div>
+{% endfor %}
+
+</div>
+{% endif %}
+
+## Top Mitwirkende
+
+Hier sind die **Top Mitwirkenden**. 👷
+
+Diese Benutzer haben [die meisten Pull Requests erstellt](help-fastapi.md#create-a-pull-request){.internal-link target=_blank}, welche *merged* wurden.
+
+Sie haben Quellcode, Dokumentation, Übersetzungen, etc. beigesteuert. 📦
+
+{% if people %}
+<div class="user-list user-list-center">
+{% for user in people.top_contributors %}
+
+<div class="user"><a href="{{ user.url }}" target="_blank"><div class="avatar-wrapper"><img src="{{ user.avatarUrl }}"/></div><div class="title">@{{ user.login }}</div></a> <div class="count">Pull Requests: {{ user.count }}</div></div>
+{% endfor %}
+
+</div>
+{% endif %}
+
+Es gibt viele andere Mitwirkende (mehr als hundert), Sie können sie alle auf der <a href="https://github.com/tiangolo/fastapi/graphs/contributors" class="external-link" target="_blank">FastAPI GitHub Contributors-Seite</a> sehen. 👷
+
+## Top Rezensenten
+
+Diese Benutzer sind die **Top-Rezensenten**. 🕵️
+
+### Rezensionen für Übersetzungen
+
+Ich spreche nur ein paar Sprachen (und nicht sehr gut 😅). Daher sind die Reviewer diejenigen, welche die [**Kompetenz zur Bestätigung von Übersetzungen**](contributing.md#translations){.internal-link target=_blank} der Dokumentation haben. Ohne sie gäbe es keine Dokumentation in mehreren anderen Sprachen.
+
+---
+
+Die **Top-Reviewer** 🕵️ haben die meisten Pull Requests von anderen überprüft und stellen die Qualität des Codes, der Dokumentation und insbesondere der **Übersetzungen** sicher.
+
+{% if people %}
+<div class="user-list user-list-center">
+{% for user in people.top_reviewers %}
+
+<div class="user"><a href="{{ user.url }}" target="_blank"><div class="avatar-wrapper"><img src="{{ user.avatarUrl }}"/></div><div class="title">@{{ user.login }}</div></a> <div class="count">Reviews: {{ user.count }}</div></div>
```

Comment:
```suggestion
<div class="user"><a href="{{ user.url }}" target="_blank"><div class="avatar-wrapper"><img src="{{ user.avatarUrl }}"/></div><div class="title">@{{ user.login }}</div></a> <div class="count">Rezenzionen: {{ user.count }}</div></div>
```

This can be translated.